crates/nargo/src/cli/mod.rs

```diff
@@ -112,7 +112,10 @@ pub fn prove_and_verify(proof_name: &str, prg_dir: &Path, show_ssa: bool) -> boo
     let tmp_dir = TempDir::new("p_and_v_tests").unwrap();
     println!("prove_with_path(_, {})", show_ssa);
     let proof_path =
-        prove_cmd::prove_with_path(proof_name, prg_dir, &tmp_dir.into_path(), show_ssa).unwrap();
+        match prove_cmd::prove_with_path(proof_name, prg_dir, &tmp_dir.into_path(), show_ssa) {
+            Ok(p) => p,
+            Err(_) => return false,
```

Comment:
Perhaps we should display the error and return false here

Reply:
Done